src/components/Item/VisualizationItem/DefaultPlugin.js

```diff
@@ -133,11 +142,15 @@ DefaultPlugin.contextTypes = {
 };
 
 DefaultPlugin.propTypes = {
+    editMode: PropTypes.bool,
```

Comment:
Is this used in `DefaultPlugin`?

Reply:
Looks like we [used it before](https://github.com/dhis2/dashboards-app/blob/master/src/components/Item/VisualizationItem/DefaultPlugin.js#L83) and I updated props, but then removed the line where it was used.

Will update props again now.